.travis.yml

```diff
@@ -27,6 +27,27 @@ jobs:
       after_success:
         # submit coverage.txt to codecov.io
         - bash <(curl -s https://codecov.io/bash)
+      
+    # YAML alias, for settings shared across the tests on windows
+    - &windows-test
+      stage: Test
+      name: "Unit test on windows"
+      os: 
+        - windows
+      init:
+      - git config --system core.longpaths true
+      go_import_path: github.com/openshift/odo
+      go: "1.13.1"
+      install:
+        - systeminfo.exe | grep '^OS'
+        - choco install make
+        - make goget-tools
+      script:
+        - export PATH="$PATH:$GOPATH/bin"
+        - make bin
+        - rm -r /tmp/*
```

Comment:
There is no `/tmp` on windows. How is this working? Is there some kind of conversion to windows paths?

Reply:
@kadel In travis minimal gnu for windows is set as a default terminal for running scripts. minimal gnu provides the similar Linux file system on windows platform. As per the gnu environment the temp file will be generated in ```/tmp``` directory. For example 

<img width="935" alt="Screen Shot 2020-03-10 at 4 16 56 PM" src="https://user-images.githubusercontent.com/19851115/76305173-a9de2a00-62ea-11ea-8c35-9f358186f20c.png">

<img width="1244" alt="Screen Shot 2020-03-10 at 4 12 23 PM" src="https://user-images.githubusercontent.com/19851115/76304789-fa08bc80-62e9-11ea-8ba6-4a743c80e089.png">

So its good practice to remove the temp files before running the actual test.